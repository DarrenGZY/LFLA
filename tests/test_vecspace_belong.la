function main()
{   
    vector v1 = [1,2,3];
    vector v2 = [2,3,4];
	vector v3 = [3,4,5];
	vector v4 = [3,5,7];
	vecspace vs = L(v1, v2 );
    print(v3@vs);
    print(v4@vs);
}
