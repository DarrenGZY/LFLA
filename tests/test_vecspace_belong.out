#!/usr/bin/python
import numpy as np
from VecSpace import *
from InSpace import *
from AffSpace import *
def main() :
    v1=np.array([1,2,3])
    v2=np.array([2,3,4])
    v3=np.array([3,4,4])
    v4=np.array([3,5,7])
    vs=VecSpace([v1, v2])
    print(vs.belongs(v3))
    print(vs.belongs(v4))

main()
